src/porepy/applications/verification_setups/mandel_biot.py

```diff
@@ -505,49 +494,46 @@ def horizontal_velocity_profile(self, x: np.ndarray, t: number) -> np.ndarray:
         return qx
 
     def flux(self, sd: pp.Grid, t: number) -> np.ndarray:
-        """Evaluate exact Darcy flux [m^3 * s^{-1}] at the face centers.
+        """Evaluate exact Darcy flux in scaled [m^3 * s^{-1}] at the face centers.
 
         Parameters:
             sd: Grid.
-            t: Time in seconds.
+            t: Time in [s].
```

Comment:
Just to be sure: Is this unscaled?

Reply:
Yes. But only because it is not yet supported by pp.Units.